### PR TITLE
Add hardhat functionality & deploy roles v2

### LINKED ISSUE #2: Fix need to run "Activate" twice
- when in a new repo, running the "Activate" command once results in an error. It needs to be run twice

### LINKED ISSUE #3: Incorrect .env names set
tenderly wizard output these as the .env variables which is incorrect

ACCESS_CONTROL_SAFE
INVESTMENT_SAFE
INVESTMENT_ROLES
ACCESS_CONTROL_ROLES
 

it must look like

ACCESS_CONTROL_SAFE_ADDRESS=0x101C568e590a803812aCfA6d1Cc9B0ED69E75aED
INVESTMENT_SAFE_ADDRESS=0x98CA8c2931a3F67aC004BCCA98DdD099f6872818
INVESTMENT_ROLES_ADDRESS=0x384e6aa9E8DcA715Ec3FC1c3DEcaA81b909f7Fa2
ACCESS_CONTROL_ROLES_ADDRESS=0x1a57a8A2647D007Ff710c556B69AB407D05368ef

### LINKED ISSUE #6: Returning WebSocket URL instead of RPC
- In some occasions a WSS URL is returned instead of an HTTPS URL for the RPC. The Wizard fails to run if in when that happens
- This is likely because and array of URLs is returned, but the position is not always the same in the array.
- Iterate the array to filter out and use the HTTP address

### LINKED ISSUE #7: ACS V2 whitelisting does not detect the correct directory of the permissions
- ACS V2 whitelisting does not detect the correct directory of the permissions
- Update the code to find the right directory

### PR DESCRIPTION
**Pull request by pripatelUK**

### CODE REVIEW

**Comment by pripatelUK** [comment]
@mikeaxle how much work is it to add the roles v1 system here? 

can we update the tool so that it explicitly says "Deploy Safe & Roles V2"

File: src/utils/constants.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    refundReceiver: hre.ethers.constants.AddressZero,
+};
+
+// MAINNET safe & roles specific addresses
+export const SAFE_MASTER_COPY_ADDR = "0x41675C099F32341bf84BFc5382aF534df5C7461a"; // @note v 1.4.0
+export const SAFE_MODULE_PROXY_FACTORY_ADDR = "0x000000000000aDdB49795b0f9bA5BC298cDda236";
+export const SAFE_PROXY_FACTORY_ADDR = "0x4e1DCf7AD4e460CfD30791CCC4F9c8a4f820ec67";
+export const ROLES_V2_MASTER_COPY_ADDR = "0x9646fDAD06d3e24444381f44362a3B0eB343D337";
+export const PERMISSIONS_LIB = "0x33D1C5A5B6a7f3885c7467e829aaa21698937597";
+export const MULTISEND_ADDR = "0x9641d764fc13c8B624c04430C7356C1C7C8102e2"; //"0x9641d764fc13c8B624c04430C7356C1C7C8102e2"; //
```
Comment (pripatelUK) [inline src/utils/constants.ts]: EEK, old multisend

**Comment by mikeaxle** [comment]
closes #6

**Comment by mikeaxle** [comment]
closes #2 
closes #3

**Pull request merged**